src/ir/selector_access_path_set.cc

```diff
@@ -0,0 +1,70 @@
+#include "selector_access_path_set.h"
+#include "selector.h"
+
+namespace raksha::ir {
+
+void SelectorAccessPathSet::AddAll(SelectorAccessPathSet other) {
```

Comment:
Should the parameter be `&` or `&&`?

Reply:
Here's why I did it this way:

`SelectorAccessPathSet` can only be silently moved. It can be copied, but only with an explicit call to `Copy`.

If the parameter `other` is populated with a move, passing it by value is very cheap. If it is populated with a copy, then this frees us from having to worry about whether the pointee of the reference is handled in a way that makes mutating it safe. As such, this is no less performant than a reference parameter but also easier to reason about.

Reply:
As in #68, I've stripped out these explicit Copy calls and let the compiler do the work.